apps/studio/tests/README.md

```diff
@@ -1,6 +1,75 @@
 # UI Testing Notes
 
-### `<Popover>` vs `<Dropdown>`
+## Rules
+
+- All tests should be run consistently (avoid situations whereby tests fails "sometimes")
+
+- Group tests in folders based on the feature they are testing. Avoid file/folder based folder names since those can change and we will forget to update the tests.
```

Comment:
I would propose moving the tests next to the file they test. The mantra I usually follow for organizing files is "files that change together, stay together":
- Easier to keep the tests up-to-date, always in your POV;
- It gives you a nice 1-to-1 mapping (this test file tests that file);
- Easy to follow guideline, avoids bike-shedding situations (cron logs belong to logs or cron feature?).

As an example, I lose a lot of time going between page files and their components (placed in `interfaces` folder), ctrl+click doesn't always work etc.

Reply:
> moving the tests next to the file they test
yeap am thinking this too for unit tests - i remember we were talking about this before RE having unit test files beside the file that they're testing